.pre-commit-config.yaml

```diff
@@ -7,15 +7,23 @@ repos:
     hooks:
     -   id: trailing-whitespace
     -   id: end-of-file-fixer
+
 -   repo: local
     hooks:
     -   id: codeql-format
         name: Fix QL file formatting
         files: \.qll?$
         language: system
         entry: codeql query format --in-place
+
     -   id: sync-files
         name: Fix files required to be identical
         language: system
         entry: python3 config/sync-files.py --latest
         pass_filenames: false
+
+    -   id: qlhelp
```

Comment:
This should probably be named `qhelp` instead of `qlhelp`?

Reply:
yep!